tests/regressions/component/CMakeLists.txt

```diff
@@ -1,25 +1,27 @@
-# Copyright (c) 2007-2017 Hartmut Kaiser
+# Copyright (c) 2007-2020 Hartmut Kaiser
 #
 # SPDX-License-Identifier: BSL-1.0
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
 set(tests
-    bulk_new_3054
     client_base_registration
     create_n_components_2323
     moveonly_constructor_arguments_1405
-    new_2848
     returned_client_2150
    )
 
 if(HPX_WITH_NETWORKING)
   set(tests ${tests}
-      create_remote_component_2334
+    bulk_new_3054
```

Comment:
These don't make sense with networking off?

Reply:
Let me have a look again. Yu might be right, they most likely perform local tests as well.

Reply:
This has been addressed by #4415